stage1/find-em.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/sh
+#
+# Find entries for the rkt ACI manifest files.
+# Run from within the CoreOS version of interest.
+#
+# Post prosessing tips:
+#   cat found.raw | egrep '^[^=]' | egrep --invert-match '[^[0-9]\.so$' | sort | uniq -c > found.uniq
+#   manual clanup
+#   cat found.uniq | sort | uniq > found.manifest
+#
+
+# TODO: get values from makefile
+#: ${BOARD:="@@BOARD@@"}
```

Comment:
Well, if makefile will generate the script, then you would still have the generated makefile, wouldn't you? :) Either way, you have to call `./configure`.

What you could do is to add a command line parameter that will have a precendence over the value in the makefile, so it won't be strictly necessary to have a generated make file first.